server/app.js

```diff
@@ -22,46 +22,46 @@ import profileRouter from "./routes/profile";
 
 var app = express();
 
-const storage = multer.diskStorage({
-  destination: (req, file, cb) => cb(null, "images"),
-  filename: (req, file, cb) => cb(null, uuidv4())
-});
+// const storage = multer.diskStorage({
+//   destination: (req, file, cb) => cb(null, "images"),
+//   filename: (req, file, cb) => cb(null, uuidv4())
+// });
 
-const fileFilter = (req, file, cb) => {
-  if (
-    file.mimetype === "image/png" ||
-    file.mimetype === "image/jpg" ||
-    file.mimetype === "image/jpeg"
-  ) {
-    cb(null, true);
-  } else {
-    cb(null, false);
-  }
-};
+// const fileFilter = (req, file, cb) => {
+//   if (
+//     file.mimetype === "image/png" ||
+//     file.mimetype === "image/jpg" ||
+//     file.mimetype === "image/jpeg"
+//   ) {
+//     cb(null, true);
+//   } else {
+//     cb(null, false);
+//   }
+// };
 
-app.use((req, res, next) => {
-  res.setHeader("Access-Control-Allow-Origin", "*");
-  res.setHeader(
-    "Access-Control-Allow-Methods",
-    "OPTIONS, GET, POST, PUT, PATCH, DELETE"
-  );
-  res.setHeader("Access-Control-Allow-Headers", "Content-Type, Authorization");
-  next();
-});
+// app.use((req, res, next) => {
+//   res.setHeader("Access-Control-Allow-Origin", "*");
+//   res.setHeader(
+//     "Access-Control-Allow-Methods",
+//     "OPTIONS, GET, POST, PUT, PATCH, DELETE"
+//   );
+//   res.setHeader("Access-Control-Allow-Headers", "Content-Type, Authorization");
+//   next();
+// });
 
 app.use(logger("dev"));
 app.use(json());
 app.use(urlencoded({ extended: false }));
 app.use(cookieParser());
 app.use("/images", express.static(path.join(__dirname, "images")));
-app.use(multer({ storage, fileFilter }).single("image"));
+// app.use(multer({ storage, fileFilter }).single("image"));
 app.use(express.static(join(__dirname, "images")));
-mongoose
-  .connect(process.env.MONGODB_URI, {
-    useNewUrlParser: true
-  })
-  .then(() => console.log("MongoDB successfully connected"))
-  .catch(err => console.log(err));
+// mongoose
```

Comment:
somehow there's 2 mongoose declarations in this file, just delete this commented out one if it works without it